core/invoker/src/main/scala/whisk/core/containerpool/docker/DockerContainer.scala

```diff
@@ -147,8 +147,8 @@ class DockerContainer(id: ContainerId, ip: ContainerIp)(implicit docker: DockerA
   /** HTTP connection to the container, will be lazily established by callContainer */
   private var httpConnection: Option[HttpUtils] = None
 
-  def suspend()(implicit transid: TransactionId): Future[Unit] = runc.pause(id)
-  def resume()(implicit transid: TransactionId): Future[Unit] = runc.resume(id)
+  def suspend()(implicit transid: TransactionId): Future[Unit] = if (runc.isInstalled) runc.pause(id) else docker.pause(id)
```

Comment:
@markusthoemmes , @rabbah any nicer scala style to replace the `if` ?

Reply:
Nope not really, the `if` is fine here.